tests/postgres_tests/test_aggregates.py

```diff
@@ -299,6 +303,16 @@ def test_string_agg_charfield(self):
         values = AggregateTestModel.objects.aggregate(stringagg=StringAgg('char_field', delimiter=';'))
         self.assertEqual(values, {'stringagg': 'Foo1;Foo2;Foo4;Foo3'})
 
+    @ignore_warnings(category=RemovedInDjango50Warning)
+    def test_string_agg_default_output_field(self):
+        try:
+            values = AggregateTestModel.objects.aggregate(stringagg=StringAgg('text_field', delimiter=';'))
+        except FieldError:
+            self.fail('default value of output_field argument in StringAgg must be a TextField')
+
+        else:
+            self.assertEqual(values, {'stringagg': 'Foo1;Foo2;Foo4;Foo3'})
```

Comment:
I think the `FieldError` is an artifact of your test iteration process, you should be able to remove the `try` statement now.